lib/sentry/event.ex

```diff
@@ -191,4 +199,23 @@ defmodule Sentry.Event do
 
   defp arity_to_integer(arity) when is_list(arity), do: Enum.count(arity)
   defp arity_to_integer(arity) when is_integer(arity), do: arity
+
+  defp is_in_app?(nil, _in_app_whitelist), do: false
+  defp is_in_app?(_, []), do: false
```

Comment:
I'm also wondering if instead of doing all that work we could rely on filepath like ruby.